src/cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeMarshal.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is a new function right that you wrote, I would just use the standard MIT licensing we use in other files.

Reply:
I would still have something like this: https://github.com/microsoft/CsWinRT/blob/9854573b9880650efd87c5be41a553d125877c92/src/WinRT.Runtime/CastExtensions.cs#L1-L2